Zenodo.js

```diff
@@ -155,7 +155,7 @@ function scrape(doc, url) {
 
 			// workaround for pre 6.0.24 versions that don't have proper item type for data
 			// if (schemaType && schemaType.includes("Dataset")) {
-			if (!datasetType == "dataset" && ZU.xpathText(doc, '//span[@class="pull-right"]/span[contains(@class, "label-default") and contains(., "Dataset")]')) {
+			if (!(datasetType == "dataset") && ZU.xpathText(doc, '//span[@class="pull-right"]/span[contains(@class, "label-default") and contains(., "Dataset")]')) {
```

Comment:
You're just going for `datasetType != "dataset"`, right? There's no need for parentheses.

Reply:
Thanks, I'm clearly a bit out of it post workshop ;)